<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件定时删除工具</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Form</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>任务列表</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0">
         <widget class="QTableWidget" name="table_list">
          <property name="editTriggers">
           <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
          </property>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>目录路径</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>文件类型</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>递归子目录</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>保留天数</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>保留大小</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>保留数量</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>定时参数</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>定时状态</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QHBoxLayout" name="task_horizontalLayout">
            <item>
             <widget class="QToolButton" name="refreshButton">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>30</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>10</width>
                <height>10</height>
               </size>
              </property>
              <property name="text">
               <string>刷 新</string>
              </property>
              <property name="popupMode">
               <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="addButton">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>新 增</string>
              </property>
              <property name="popupMode">
               <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="editButton">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>修 改</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="delButton">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>删 除</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="task_horizontalLayout2">
            <item>
             <widget class="QToolButton" name="fmdButton">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>手动删除</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="fmdtButton">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>手动删除测试</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="startTaskButton">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>定时删除</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="autoStartCheckBox">
          <property name="text">
           <string>开机启动</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="schedule_tag">
       <attribute name="title">
        <string>任务定时</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="sched_horizontalLayout">
          <item>
           <widget class="QToolButton" name="sched_refreshButton">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>刷 新</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="sched_stopButton">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>暂 停</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="sched_delButton">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>删 除</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="sched_startButton">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>启 动</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QTableWidget" name="sched_table">
          <property name="editTriggers">
           <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
          </property>
          <column>
           <property name="text">
            <string>任务名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>定时参数</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>运行状态</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>下次运行时间</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>任务日志</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="8" column="0" colspan="3">
         <widget class="QTextEdit" name="logEdit"/>
        </item>
        <item row="1" column="0" colspan="3">
         <layout class="QHBoxLayout" name="log_horizontalLayout">
          <item>
           <widget class="QToolButton" name="log_clearButton">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>清理日志</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
